src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.json

```diff
@@ -1,7 +1,12 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Debug"
+      "Default": "Information"
+    },
+    "ApplicationInsights": {
+      "LogLevel": {
+        "Default": "Information"
```

Comment:
This conflicts with the following line already in Startup.cs
`builder.AddFilter<ApplicationInsightsLoggerProvider>("", HostingEnvironment.IsDevelopment() ? LogLevel.Debug : LogLevel.Warning);`

Reply:
Resolved